modules/react/package.json

```diff
@@ -58,7 +58,7 @@
     "@deck.gl/mapbox": ">=9.1.0",
     "@deck.gl/react": ">=9.1.0",
     "@deck.gl/widgets": ">=9.1.0",
-    "@hubble.gl/core": "1.4.0",
+    "@hubble.gl/core": "^2.0.0-alpha.3",
```

Comment:
We should work on upgrading the example deps, however none of them are bundled into the released source. Therefore, this is the only change we'd actually need to resolve the npm issue.